localization/english/culture/ctp_titles_l_english.yml

```diff
@@ -1146,33 +1146,34 @@ empire_feudalclan_serbian:0 "Carstvo"
 
 #vlach
 
-baron_feudalclan_male_vlach:0 "Baron"
-baron_republic_male_vlach:0 "Primar" 
-baron_feudalclan_female_vlach:0 "Baronesă" 
-baron_republic_female_vlach:0 "Primăreasă"
-barony_feudalclan_vlach:0 "Baronie"
+baron_feudalclan_male_vlach:0 "Castelan"
+baron_republic_male_vlach:0 "Jude" # Used in Wallachia. In Moldova it was "Șoltuz" (romanianised variant of the transylvanian saxon "schulteiss" (mayor))
+baron_feudalclan_female_vlach:0 "Castelană"
+baron_republic_female_vlach:0 "Jude" # I'm not aware of any feminine variant but the original one is pretty gender-neutral
+barony_feudalclan_vlach:0 "Cetate"
 barony_republic_vlach:0 "Oraș"
-count_feudalclan_male_vlach:0 "Conte"
-count_tribal_male_vlach:0 "Voievod" 
-count_feudalclan_female_vlach:0 "Contesă"   
-count_feudalclan_male_vlach_independent:0 "Principe"  
-count_feudalclan_female_vlach_independent:0 "Principesă" 
-county_feudalclan_vlach:0 "Comitat"  
-county_tribal_vlach:0 "Metropolă" 
-county_feudalclan_vlach_independent:0 "Principalitate"  
-duke_feudalclan_male_vlach:0 "Duce" 
-duke_tribal_male_vlach:0 "Mare Voievod"  
-duke_feudalclan_male_vlach_independent:0 "Principe"  
-duke_feudalclan_female_vlach:0 "Ducesă" 
-duke_feudalclan_female_vlach_independent:0 "Principesă"  
-duchy_feudalclan_vlach:0 "Ducat"  
-duchy_feudalclan_vlach_independent:0 "Principalitate" 
-king_feudalclan_male_vlach:0 "Rege"  
-king_feudalclan_female_vlach:0 "Regină" 
-kingdom_feudalclan_vlach:0 "Regat"  
-emperor_feudalclan_male_vlach:0 "Împărat" 
-emperor_feudalclan_female_vlach:0 "Împărăteasă" 
-empire_feudalclan_vlach:0 "Imperiu" 
+count_feudalclan_male_vlach:0 "Ban" # Banul de județ: The vassal of a "Mare Ban", ruler of a "județ"
+count_tribal_male_vlach:0 "Cneaz"
+count_tribal_female_vlach:0 "Cnează"
+count_feudalclan_female_vlach:0 "Bană"
+count_feudalclan_male_vlach_independent:0 "Cneaz"
+count_feudalclan_female_vlach_independent:0 "Cnează"
+county_feudalclan_vlach:0 "Județ"
+county_tribal_vlach:0 "Cnezat"
+county_feudalclan_vlach_independent:0 "Cnezat"  
+duke_feudalclan_male_vlach:0 "Banat"
```

Comment:
Clearly a mistake here: Banat should've been duchy or county - not a person.

Reply:
It is, thank you